core/views.py

```diff
@@ -332,5 +321,10 @@ def get(self, request, *args, **kwargs):
                 html_email_template_name=self.email_template,
                 extra_email_context=self.extra_email_context,
             )
+
+
+        if request.GET.get("rbf") is None and int(request.GET["status"]) == 0:
```

Comment:
Sometimes payment directly confirm (status-1 or 2) without giving status 0 callback. We need to handle this as well.